setup-jest.js

```diff
@@ -0,0 +1,4 @@
+import { TextEncoder, TextDecoder } from 'util';
+
+global.TextEncoder = TextEncoder;
```

Comment:
Mhh, can you explain why is this needed?

Reply:
That's to get rid of these errors when running the tests:
```
    ReferenceError: TextEncoder is not defined

      1 | import { createContext, useState, useMemo, useContext, ReactNode } from 'react';
      2 | import { initialSettableChainData, initialChainData, SettableChainData } from './state';
    > 3 | import { useAccount, useNetwork } from 'wagmi';
        | ^
      4 | import { getDaoAddresses, updateNetworkName } from '../contracts';
      5 | import { useEthersProvider, useEthersSigner } from './adapters';
      6 | import { ethers } from 'ethers';

      at Object.<anonymous> (node_modules/viem/utils/encoding/toHex.ts:226:31)
      at Object.<anonymous> (node_modules/viem/utils/encoding/toBytes.ts:8:1)
      at Object.<anonymous> (node_modules/viem/utils/encoding/fromHex.ts:12:1)
      at Object.<anonymous> (node_modules/viem/chains/optimism/formatters.ts:4:1)
      at Object.<anonymous> (node_modules/viem/chains/definitions/base.ts:2:1)
      at Object.<anonymous> (node_modules/viem/chains/index.ts:10:1)
      at Object.<anonymous> (node_modules/@wagmi/core/dist/chunk-4E64EZ7V.js:2:1)
      at Object.<anonymous> (node_modules/@wagmi/core/dist/index.js:1:1)
      at Object.<anonymous> (node_modules/wagmi/dist/index.js:7:1)
      at Object.<anonymous> (src/chain-data/context.tsx:3:1)
      at Object.<anonymous> (src/chain-data/index.ts:2:1)
```